openFrameworks.i

```diff
@@ -1143,6 +1215,8 @@ class fstream {};
 
 // DIFF: ofURLFileLoader.h:ignoring ofHttpResponse ofBuffer operator
 %ignore ofHttpResponse::operator ofBuffer&();
+%ignore ofSaveURLTo(const string& url, const std::filesystem::path& path);
```

Comment:
Yes, you're right, they should be kept.  In my particular case I didn't need these two methods.